tests/dream/geant4_reduction_test.py

```diff
@@ -187,11 +183,36 @@ def test_pipeline_can_compute_dspacing_result_with_integrated_monitor_norm(
         params_for_det, run_norm=powder.RunNormalization.monitor_integrated
     )
     workflow = powder.with_pixel_mask_filenames(workflow, [])
-    result = workflow.compute(IofDspacing)
+    result = workflow.compute(IofDspacing[SampleRun])
     assert result.sizes == {'dspacing': len(params[DspacingBins]) - 1}
     assert sc.identical(result.coords['dspacing'], params[DspacingBins])
 
 
+def test_pipeline_normalizes_and_subtracts_empty_can_as_expected(
+    workflow: sciline.Pipeline,
+) -> None:
+    workflow[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.drop
+    workflow = powder.with_pixel_mask_filenames(workflow, [])
+    results = workflow.compute(
+        [
+            EmptyCanSubtractedIofDspacing[SampleRun],
+            FocussedDataDspacing[SampleRun],
+            FocussedDataDspacing[VanadiumRun],
+            FocussedDataDspacing[BackgroundRun],
+        ]
+    )
+    result = results[EmptyCanSubtractedIofDspacing[SampleRun]]
+
+    sample = results[FocussedDataDspacing[SampleRun]]
+    empty_can = results[FocussedDataDspacing[BackgroundRun]]
+    vanadium = results[FocussedDataDspacing[VanadiumRun]]
+
+    expected = powder.correction.normalize_by_vanadium_dspacing(
+        sample.bins.concatenate(-empty_can), vanadium, UncertaintyBroadcastMode.drop
+    )
+    sc.testing.assert_allclose(result, expected)
```

Comment:
Ok, but I thought we wanted to avoid always running a full workflow in every "unit" test, due to growing run times... 😬

Reply:
Good point. This was easy enough to change.